code/clue.js

```diff
@@ -1,59 +1,353 @@
+  document.getElementById('weaponHeadline').style.display = 'none';
+  document.getElementById('weaponTitle').style.display = 'none';
+  document.getElementById('weaponContent').style.display = 'none';
+  document.getElementById('weaponDecoration').style.display = 'flex'
+  document.getElementById('weaponCard').style.background = 'black'
+  document.getElementById('weaponDecoration').style.border = '8px solid #575A53'
+}
+
+const showRoomLoader = () => {
+  document.getElementById('roomLoader').style.display = 'flex';
+  document.getElementById('roomRoundLoader').classList.add('room-loader');
+  document.getElementById('roomSmallLoader').style.borderRight = '5px solid #a8979f';
+  document.getElementById('roomLoaderStick').style.backgroundColor = '#a8979f';
+  document.getElementById('roomHeadline').style.display = 'none';
+  document.getElementById('roomTitle').style.display = 'none';
+  document.getElementById('roomContent').style.display = 'none';
+  document.getElementById('roomDecoration').style.display = 'flex';
+  document.getElementById('roomCard').style.background = 'black';
+  document.getElementById('roomDecoration').style.border = '8px solid #a8979f';
+}
+  
+const generateKiller = () => {
+  // This will randomly select a killer from the suspects, add that to the mystery object and that killers favourite weapon to the moreWeapons array. 
   mystery.killer = randomSelector(suspects)
+  killerWeapons.fill(mystery.killer.favouriteWeapon);
+  moreWeapons = weapons.concat(killerWeapons)
+ 
+  // Hides the loader and shows killer title and content 
+  document.getElementById('killerLoader').style.display = 'none';
+  document.getElementById('killerTitle').style.display = 'block';
+  document.getElementById('killerContent').style.display = 'block';
+
+  //Changes the text and image depending on the mystery values
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerAge').innerHTML = 'Age: ' + mystery.killer.age
+  document.getElementById('killerOccupation').innerHTML = 'Occupation: ' + mystery.killer.occupation
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+  //Styles the killer card
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerTitle').style.display = 'block'
+  document.getElementById('killerDecoration').style.display = 'block'
+  document.getElementById('killerDecoration').style.border = '0px'
+  document.getElementsByTagName('img')[0].style.borderRadius = '50%'
+  const classKillerContent = document.getElementsByClassName('killer-content')[0];
```

Comment:
Seems like you are looking for one specific element, why not use `getElementById` here?